src/node_v8.cc

```diff
@@ -295,6 +295,154 @@ static void SetHeapStatistics(JSONWriter* writer, Isolate* isolate) {
   writer->json_arrayend();
 }
 
+static v8::Local<v8::Object> ConvertHeapStatsToJSObject(
+    v8::Isolate* isolate, const cppgc::HeapStatistics& stats) {
+  Local<Context> context = isolate->GetCurrentContext();
+  // Space Statistics
+  v8::LocalVector<v8::Value> spaceStatisticsArray(isolate);
```

Comment:
```suggestion
  v8::LocalVector<v8::Value> space_statistics_array(isolate);
```

Node.js uses snake cases for variable names (which is also what V8 does), see https://github.com/nodejs/node/blob/main/doc/contributing/cpp-style-guide.md#snake_case-for-local-variables-and-parameters